inc/widget/latest_news.php

```diff
@@ -1,13 +1,13 @@
 <?php
 namespace W3TC;
 
-if ( !defined( 'W3TC' ) )
+if ( ! defined( 'W3TC' ) ) {
 	die();
-
+}
 ?>
-<p class="widget-loading hide-if-no-js {nonce: '<?php echo wp_create_nonce( 'w3tc' ); ?>'}">
-	<?php echo __( 'Loading&#8230;' ) ?>
+<p class="widget-loading hide-if-no-js {nonce: '<?php echo esc_attr( wp_create_nonce( 'w3tc' ) ); ?>'}">
+	<?php esc_html_e( 'Loading&#8230;', 'w3-total-cache' ); ?>
```

Comment:
The HTML entity is being escaped when it should be parsed.  Use `printf()` and `esc_html__()`.